javalib/src/main/scala/java/math/BigDecimal.scala

```diff
@@ -195,7 +195,7 @@ object BigDecimal {
   }
 
   private[math] def newArrayOfPows(len: Int, pow: Int): Array[Long] =
-    new Array[Long](len - 1).scanLeft[Long, Array[Long]](1)((z, e) => z * pow)
+    new Array[Long](len - 1).scanLeft(1L)((z, e) => z * pow)/* temporary, see scala/collection-strawman#544 */.toArray
```

Comment:
`scanLeft` now takes only one type parameter instead of two.

Reply:
The “temporary” was about the `.toArray` call, which has been fixed by https://github.com/scala/scala/pull/6506 in the meantime.